cmd/crc/cmd/util.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/code-ready/crc/pkg/crc/daemonclient"
+	crcErr "github.com/code-ready/crc/pkg/crc/errors"
+)
+
+func checkIfMachineMissingWithDaemon(client *daemonclient.Client) error {
+	exists, err := client.APIClient.Exists()
```

Comment:
I am not sure if we can assume `client *daemonclient.Client` has a valid value.
Perhaps safeguard this with an additional check?